### DISCUSSION TITLE
Migração React Native > Flutter

### DISCUSSION THREAD

**Discussion by fabiojansenbr** [Flutterando/forum#55] [Perguntas e Respostas]
Pessoal,
Sabemos que é possível a partir de um app nativo, chamar um app em Flutter, de modo que o usuário nem perceba esta mudança.

É possível fazer o mesmo com o React Native? Tenho um app que será migrado pra Flutter, e gostaria talvez de ir liberando as features aos poucos, de modo que o usuário não precise ter as duas versões do app instaladas. Tem como? Qual a linha de pesquisa pra isso? Só encontrei pra Nativo.

**Comment by eduardoflorence** [comment]
@fabiojansenbr, acredito que para essa migração, cada feature feita em flutter terá que ser um app diferente, onde você irá fazer o build para web (flutter web) e pode carregar separadamente nas suas rotas do React.
Nessa resposta do stackoverflow, mostra uma alternativa para carregar na rota:
https://stackoverflow.com/a/70935986/19838611
